nishimweelysee/Elysee-CapstoneProject
# Ft list questions

#### What does this PR do?
>- This PR contain a Branch of List of Questions/Queries this are a contacts messages sent by the users to the blog owner.
>
#### Description of Task to be completed
>
>- view phone number
>- View Email Address
>- View Full name
>- View Address
>- View Message Sent
>
#### How should this be manually tested?
>
>Open a website and login as an admin 
>Go to Contact Tab
>You will see the list of all messages sent.
>
#### Any background context you want to provide?
>
#### What are the relevant Trello story?
>FT(List of all Queries or Questions ) use this link to see my trello card => https://trello.com/c/8a5JfQKk
#### Screenshots (if appropriate)
>
![image](https://user-images.githubusercontent.com/56052588/92355516-b3531e00-f0e4-11ea-8028-b78ba718d000.png)

![image](https://user-images.githubusercontent.com/56052588/92355561-c82fb180-f0e4-11ea-9b49-5b7fc963f939.png)


Use this picture to see,and for the feedback you gave me i have tried to put that tab basing on numbers when one full the next messages will go to the next tab until finished.
